kani-compiler/src/intrinsics.rs

```diff
@@ -0,0 +1,359 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Single source of truth about which intrinsics we support.
+
+// Enumeration of all intrinsics we support right now, with the last option being a catch-all. This
+// way, adding an intrinsic would highlight all places where they are used.
+#[allow(unused)]
+#[derive(Clone, Copy, Debug)]
+pub enum Intrinsic<'a> {
```

Comment:
Have you considered using the [symbols](https://github.com/rust-lang/rust/blob/d3a393932eeafa4638ae22f5ecbc38bf38760d0e/compiler/rustc_span/src/symbol.rs#L128) from `rustc_span` instead?

Reply:
I don't think this would make it easier because Kani does not support every single intrinsic. Using symbols also would not allow us to split intrinsics by their prefix